libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.ts

```diff
@@ -302,6 +302,12 @@ export class SkyAutocompleteComponent implements OnDestroy, AfterViewInit {
   @Output()
   public searchAsync = new EventEmitter<SkyAutocompleteSearchAsyncArgs>();
 
+  /**
+   * @internal
+   */
+  @Output()
+  public autocompleteOpenChange = new EventEmitter<boolean>();
```

Comment:
Since the underlying property would typically be named `open` (or `isOpen`), we can go with `openChange` for all of these (the prefix of "autocomplete", "lookup", "calendar", etc. should be inferred from the component's name).
```
<sky-lookup
  (openChange)="onLookupOpenChange($event)"
></sky-lookup>
```

Reply:
Good call. Updated.